autogen/Hardware.m

```diff
@@ -0,0 +1,20 @@
+classdef Hardware
+    %HARDWARE Represents target hardware attributes
+    
+    properties
+        deviceFamily
+        device
+    end
+    enumeration
+        Audiomini('Cyclone V', '5CSEBA6U23I7')
+        Audioblade('Arria 10', '10AS066H2F34I1HG')
```

Comment:
Are we doing TitleCase for enumerations? If so, perhaps these should be `AudioMini` and `AudioBlade`?

Reply:
We don't really have anything defined for enumerations, but [Matlab](https://www.mathworks.com/help/matlab/matlab_oop/enumerations.html) uses TitleCase in their reference material that I used to make the enumerations. But yeah, I can update that to `AudioMini` and `AudioBlade`, I just keep forgetting their names aren't a single word.

Reply:
It doesn't help that I've used their names as all-lowercase single words elsewhere...